code/game/objects/items/devices/taperecorder.dm

```diff
@@ -278,6 +281,16 @@
 				print_transcript()
 			if("Eject")
 				eject(user)
+			if("Wipe")
+				wipeproc()
+
+
+/obj/item/taperecorder/proc/wipeproc()
+	mytape.used_capacity = 0
+	mytape.storedinfo = new
+	mytape.timestamp = new
+	to_chat(usr, "<span class='notice'>You wipe this side of the tape entirely.")
```

Comment:
Pass the user in directly--we don't use `usr` unless we have to due to its inconsistencies.

Reply:
Then, use balloon alerts instead of to_chat, which we use for quick actions that need feedback but not archival.

```suggestion
	balloon_alert(user, "wiped tape")
```